components/domains/renewal.tsx

```diff
@@ -61,8 +58,9 @@ const Renewal: FunctionComponent<RenewalProps> = ({ groups }) => {
   const [termsBox, setTermsBox] = useState<boolean>(true);
   const [renewalBox, setRenewalBox] = useState<boolean>(true);
   const [walletModalOpen, setWalletModalOpen] = useState<boolean>(false);
-  const [salts, setSalts] = useState<string[] | undefined>();
-  const [metadataHashes, setMetadataHashes] = useState<string[] | undefined>();
```

Comment:
Why we wouldn't need several metdatahashes anymore ? Is there a particular reason ? @Th0rgal might need you opinion here too

Reply:
I asked @Th0rgal who told me one metahash was enough, if we want to use different metahash then I'd have to update the endpoint in the server to return an array with a metahash for each domain owned.

Reply:
Yes one metadata should not be an issue, the endpoint must just return the last one